src/drivers/mct/cime_config/buildnml

```diff
@@ -387,6 +387,12 @@ def buildnml(case, caseroot, component):
         if "aquaplanet" in cam_config_opts:
             infile_text = "aqua_planet = .true. \n aqua_planet_sst = 1"
 
+    if case.get_value('COMP_ATM') == 'eam':
+        # cam is actually changing the driver namelist settings
+        cam_config_opts = case.get_value("CAM_CONFIG_OPTS")
+        if "aquaplanet" in cam_config_opts:
+            infile_text = "aqua_planet = .true. \n aqua_planet_sst = 1"
+
```

Comment:
Unless I'm missing something, it looks like the bodies of these conditionals are identical. Is there a reason you copied this block of code rather than changing the conditional above to something like `if case.get_value('COMP_ATM') in ['cam', 'eam']`?

Reply:
They are currently identical but we might change CAM_CONFIG_OPTS to EAM_CONFIG_OPTS later so made the full copy.

Reply:
It seems like that could still be generalized with variable substitution rather than hard-coding "CAM_CONFIG_OPTS", but since this isn't too much duplicated code and I could see how these might need to diverge even further in the future, I don't feel strongly about this myself.

Reply:
I'll make the change you requested since there probably is a better solution to that.

Reply:
What the right syntax for that if clause?  I'm getting a syntax error with what you wrote.

Reply:
I forgot the colon at the end.